scripts/install-helpers/baremetal-coco/install.sh

```diff
@@ -247,6 +278,40 @@ function deploy_intel_device_plugins() {
     echo "Intel Device Plugins operator | deployment finished successfully"
 }
 
+function deploy_intel_dcap() {
+    echo "Intel DCAP | starting the deployment"
+
+    pushd intel-dcap || return 1
+    oc apply -f ns.yaml || return 1
+
+    oc project intel-dcap
+    oc adm policy add-scc-to-user privileged -z default
```

Comment:
does the dcap need to run in a privileged mode?

Reply:
Yes, it does.